src/modeler/loadFiles/readDataSeries.cpp

```diff
@@ -34,7 +34,7 @@ std::unique_ptr<Optimisation::LinearProblemApi::ILinearProblemData> loadDataSeri
         Optimisation::LinearProblemDataImpl::DataSeriesRepository dataSeriesRepository = IO::
           Inputs::DataSeriesCsvImporter::DataSeriesRepoImporter::importFromDirectory(
             studyPath / "input" / "data-series",
-            "\t");
+            "\t ");
```

Comment:
Is this change intended ?

Reply:
Intended but should be discussed.
Currently the specification is that tsv use tab as separator. If one use a space it doesn't works, much less several spaces.
As a developer it's a pain to use tabs because my tools are configured to automaticaly insert spaces.

Reply:
OK, so either tabs or spaces are accepted as CSV separators ? I mean that's good to me but it needs to be documented